examples/advanced/rag/generation/README.md

```diff
@@ -0,0 +1,2 @@
+# RAG with federated context retrieval
+This example shows how to use [NVIDIA FLARE](https://nvidia.github.io/NVFlare) for RAG with federated context retrieval, collecting context from multiple clients for generation tasks.
```

Comment:
should we add this in another PR

Reply:
removed this part while keeping the folder structure for future additions